test/Game-test.js

```diff
@@ -0,0 +1,73 @@
+// what exactly is currentRound??
```

Comment:
Need to clean all this up. But first I need to figure out the TDD for Game class